test/consul/config.hcl

```diff
@@ -1,6 +1,20 @@
-client_addr = "10.55.55.10"
+client_addr = "0.0.0.0"
```

Comment:
I assume that this change is somehow necessary for it to successfully make outbound connections? I don't quite follow why.

Reply:
Another working statement would be `client_addr = "10.55.55.10 10.77.77.10 10.88.88.10"` e.g.
the consul client is listening on consulnet, bluenet, and rednet. This also matters for TLS SAN matching purposes from what I remember back in staging/prod consul.

Reply:
This instructs Consul to listen on all interfaces. Previously, we were only listening on one interface and vecause of how Docker's network bridge works we could actually continue to listen on just `10.55.55.10` and our stack would work just fine. Unfortunately, what did break with this change was the ability to expose `10.55.55.10:8500` on macOS. By swapping it to listen on all of the interfaces Consul is now listening on the interface exposed to the parent hypervisor on macOS (only when instructed by adding `ports: 8500:8500` in the docker compose file.

Reply:
:psyduck-why: